resources/js/visual-editor/blocks/icon/icon.css

```
@@ -0,0 +1,37 @@
+.wp-block-artisanpack-icon {
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    line-height: 0;
+    max-width: 100%;
+}
+
+.wp-block-artisanpack-icon svg {
+    width: 100%;
+    height: 100%;
+    fill: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)